code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -12,9 +12,12 @@
 /obj/item/organ/cyberimp/bci/Initialize(mapload)
 	. = ..()
 
+	var/obj/item/integrated_circuit/circuit = new(src)
+	circuit.add_component(new /obj/item/circuit_component/equipment_action/bci(null, "One"))
```

Comment:
why do you make them start with a circuit? it makes them unable to have cell inserted

Reply:
also why does it start with bci action preequipped

Reply:
they were like that before, we changed that specifically so players can just just ones they need

Reply:
with your current thing, you 
- print BCI shell
- assemble BCI shell
- take circuit out of BCI shell
- put battery in BCI shell
- put circuit back in BCI shell

Reply:
37a0a42